### ISSUE TITLE
Certificate authentication not working when deployed to azure app service RestSharp/108.0.3.0

### ISSUE THREAD

**Issue by chandan1001** [restsharp/RestSharp#1985]
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel. at System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult) at System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar) 

error appears when app is deployed to azure app service. It is working locally on dev PCs.  I've posted more details on SO. 
```

var x509 = new X509Certificate2();
x509.Import(certificateBytes, certificatePassword, X509KeyStorageFlags.DefaultKeySet);
certificateCollection = new X509Certificate2Collection { x509 };

var options = new RestClientOptions(new Uri(serverBaseUrl))
            {
                MaxTimeout = (int)TimeSpan.FromMinutes(5).TotalMilliseconds,
            };

if (certificateCollection != null)
{
    options.ClientCertificates = certificateCollection;
}

var client = new RestClient(options);
```

[StackOverflow question link](https://stackoverflow.com/questions/74802265/restsharpv107-clientcertificate-authentication-broken-in-azure-the-request-was)

**Comment by Kevin-TokyWoky** [comment]
Hello @chandan1001,

Looks like we have the same issue, however it seems it is not related to RestSharp.
We have removed RestSharp and used the default HttpClient, and still have the same error.
It makes sense, since RestSharp is using the underlying HttpClient

This is what we do using HttpClient
```
var httpClientHandler = new HttpClientHandler
{
    SslProtocols = SslProtocols.Tls12
};
httpClientHandler.ClientCertificateOptions = ClientCertificateOption.Manual;
httpClientHandler.ClientCertificates.Add(new X509Certificate2(pfxCertificate, CertificatePassword));
```

Do you know what's going on?
Have you solved this?

**Comment by Kevin-TokyWoky** [comment]
@chandan1001 We have found out what the problem was.
I have replied to the SO question you linked, here is my asnwer: https://stackoverflow.com/a/75457827/8807586

TLDR; add a `WEBSITE_LOAD_CERTIFICATES=*` to your App Service settings.
Documentation there: https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-certificate-in-code#make-the-certificate-accessible
